### PR TITLE
Set the foundations for a new design w/o bootstrap

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

**Comment by davidbgk** [comment]
Now with spaces 🎉

File: repondeur/zam_repondeur/resources.py [hunk 1]
```diff
@@ -149,6 +150,45 @@ def model(self) -> Amendement:
...
+
+    @property
+    def parent(self) -> LectureResource:
+        return cast(LectureResource, self.__parent__)
+
+
+class SpaceResource(Resource):
+    def __init__(self, name: str, parent: Resource) -> None:
+        super().__init__(name=name, parent=parent)
+        self.name = name
```
Comment (ronnix) [inline repondeur/zam_repondeur/resources.py]: Do we need the `name` attribute ? If not we can drop the `__init__`.

**Comment by davidbgk** [comment]
Now without spaces 🤷‍♂️

**Comment by davidbgk** [comment]
Now without teams 👨‍🚒

**Pull request merged**